src/Service/Config/ScriptService.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace MyPa\Shopware\Service\Config;
+
+class ScriptService
+{
+    /**
+     * @return string
+     */
+    private function getDeliveryOptionsVersion() {
+        return '5.8.0';
```

Comment:
we zitten op 5.9.0 

https://github.com/myparcelnl/delivery-options/releases

Wat je ook kan doen is met unpkg automatisch de laatste versie op major 5 pakken:

> `<package>@^5`

Mocht je dat willen doen... Let er wel op dat de `^` url encoded moet zijn, gebeurt niet altijd vanzelf